web/js/components/measure-tool/measure-menu.js

```diff
@@ -68,11 +73,6 @@ const MeasureMenu = function () {
     dispatch(changeUnits(units));
```

Comment:
Can we write out this dispatch call to it's own function as we did for onToggle()?

```
const changeUnits = (units) => { dispatch(changeUnitsAction(units)); };
```
And then just
```
changeUnits(units);
```

Reply:
Good catch. I just updated it to wrap a function around the dispatch.